docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,358 @@
+---
+docs: DOCS-000
+title: Compile NAP WAF policies using NGINX Instance Manager
+toc: true
+weight: 300
+---
+
+## Overview
+
+This guide describes how to use F5 NGINX Instance Manager to compile NGINX App Protect WAF Policies for use with NGINX Ingress Controller.
+
+NGINX App Protect WAF uses policies to configure which security features are set. When these policies are changed, they need to be compiled so that the engine can begin to use them. Compiling policies can take a large amount of time and resources, depending on the size. The preferred way to do this is with NGINX Instance Manager. This will reduce the impact on a NGINX Ingress Controller deployment.
```

Comment:
Pushing present tense and second person

```suggestion
NGINX App Protect WAF uses policies to configure which security features are set. When these policies are changed, they need to be compiled so that the engine can begin to use them. Compiling policies can take a large amount of time and resources. You can do this with the NGINX Instance Manager. This reduces the impact on a NGINX Ingress Controller deployment.
```